lib/compat/wordpress-6.4/rest-api.php

```diff
@@ -27,3 +27,18 @@ function gutenberg_register_global_styles_revisions_endpoints() {
 	$global_styles_revisions_controller->register_routes();
 }
 add_action( 'rest_api_init', 'gutenberg_register_global_styles_revisions_endpoints' );
+
+/**
+ * Updates `wp_global_styles` to show in rest api. This way it will appear in the post types response.
+ *
+ * @param array  $args Array of arguments for registering a post type.
+ * @param string $post_type Post type key.
+ */
+function gutenberg_update_wp_global_styles_post_type( $args, $post_type ) {
+	if ( 'wp_global_styles' === $post_type ) {
+		$args['show_in_rest'] = true;
```

Comment:
If this PR ever makes it, let's revert this and move to whichever PR updates the global styles revisions to use the new API.

Reply:
I want to understand better why you had to do that. The global styles AFAIK already have a dedicated endpoint, was that endpoint not good enough?